src/OrchardCore.Modules/Orchard.Resources/Content/Scripts/popper.js

```diff
@@ -0,0 +1,2444 @@
+/*
```

Comment:
Bootstrap suggest 1.11.0 for the beta. Even 1.11.1 didn't seem to work with Bootstrap.